### PR TITLE
CVE-2015-1260 and CVE-2014-7934

### PR DESCRIPTION
**Pull request by scioffi**

### CODE REVIEW

File: cves/CVE-2014-7934.yml [hunk 1]
```diff
@@ -32,7 +32,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  A user was able to crash Chrome by executing a complex series of JavaScript 
+  commands. This crash was caused by a used-after-free issue. This was crash dump
```
Comment (mjschuetze102) [inline cves/CVE-2014-7934.yml]: > "This was crash dump ocurred when in certain cases, an object was deleted, but was then referenced to again after its delition."

Could you clean up this sentence, both of the misspellings and remove the was at the beginning.

File: cves/CVE-2014-7934.yml [hunk 2]
```diff
@@ -64,7 +68,11 @@ fixes:
   :note: ''
```
Comment (mjschuetze102) [inline cves/CVE-2014-7934.yml]: There is a duplicate commit hash. Could you also write a little description of the fixes that occurred between each commit.

File: cves/CVE-2014-7934.yml [hunk 3]
```diff
@@ -183,14 +196,20 @@ lessons:
...
     applies: 
     note: 
   complex_inputs:
-    applies: 
-    note: 
+    applies: true
+    note: |
+      This vulnerability is not something the normal user would encounter. The person who 
+      discovered this vulnerability had to execute a very complex layer of XML, which is 
+      a tree-based structure to cause this memory issue.
```
Comment (mjschuetze102) [inline cves/CVE-2014-7934.yml]: > ... a tree-based structure[,] to cause this memory issue.

Add a comma to  make the sentence more clear.

File: cves/CVE-2014-7934.yml [hunk 4]
```diff
@@ -137,10 +149,11 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: |
+    This CVE/bug originates to the original WebHook fork from Apple.
```
Comment (mjschuetze102) [inline cves/CVE-2014-7934.yml]: What is WebHook. This is the first time it is mentioned. Did you mean WebKit? If not, I would write some clarification here.

File: cves/CVE-2015-1260.yml [hunk 5]
```diff
@@ -204,5 +215,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: |
+    The mistake for this vulenrability was made when an engineer decided to
```
Comment (mjschuetze102) [inline cves/CVE-2015-1260.yml]: > vulenrability

Correct spelling

File: cves/CVE-2015-1260.yml [hunk 6]
```diff
@@ -65,7 +64,9 @@ fixes:
   :note: ''
 vccs:
 - commit: e4af29985a69116a4d37bf7c8130336c63542adc
-  note: 
+  note: |
+    The developer was fixing some cases in a switch statement for 
+    an unrelated bug.
 upvotes_instructions: |
```
Comment (Jss7268) [inline cves/CVE-2015-1260.yml]: Add 1 upvote for this cve

File: cves/CVE-2014-7934.yml [hunk 7]
```diff
@@ -64,7 +68,11 @@ fixes:
   :note: ''
 - :commit: abbb4880590d4b4934d0a2db3251f253733ae362
   :note: ''
-vccs: []
+vccs: 
+  - commit: 56d3dae3e51988ba04982b46b31eae658bcdb784
+    note: |
+      "Original if statement was created by Apple before WebKit was 
+      forked by Google"
 upvotes_instructions: |
```
Comment (Jss7268) [inline cves/CVE-2014-7934.yml]: Add 2 upvotes for this cve

File: cves/CVE-2015-1260.yml (hunk 5 above)
Comment (scioffi) [reply cves/CVE-2015-1260.yml]: Glad you agree.

File: cves/CVE-2014-7934.yml (hunk 4 above)
Comment (scioffi) [reply cves/CVE-2014-7934.yml]: Downside of having a computer with autocorrect 😜

File: cves/CVE-2014-7934.yml (hunk 7 above)
Comment (mesternefeld) [reply cves/CVE-2014-7934.yml]: Add 2 upvotes from me! Interesting that it existed before

File: cves/CVE-2014-7934.yml (hunk 2 above)
Comment (scioffi) [reply cves/CVE-2014-7934.yml]: Yeah, my fixes came like that. Maybe @andymeneely can clarify why they fixes 1 and 3 are identical commits.

File: cves/CVE-2015-1260.yml [hunk 8]
```diff
@@ -33,9 +33,8 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: Attackers can execute arbitrary Java Script code upon completion of a
-  getUserMedia request. The fix prevented the RenderFrame object from being destroyed
-  within the getusermedia subroutine, thus eliminating the "free" in the "user-after-free"
-  vulnerability. During testing and fixing, this was deemed medium severity.
+  getUserMedia request. The Object in question was being destroyed within the
```
[the remark this answers is not in the document]
Comment (scioffi) [reply cves/CVE-2015-1260.yml]: @Jss7268  Good catch.

File: cves/CVE-2015-1260.yml [hunk 9]
```diff
@@ -126,8 +129,10 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit: 
-    note: 
+  - commit: 98b51d8ead31a00a78f59c6bc7c187ee10f7a43e
+    note: |
+      This commit patches an earlier patch and even states that this fix does not
```
[the remark this answers is not in the document]
Comment (scioffi) [reply cves/CVE-2015-1260.yml]: @Jss7268 "This" is referring to the commit which is the definite article in this sentence.

**Pull request merged**